src/status_im/contexts/wallet/wallet_connect/events.cljs

```diff
@@ -236,16 +261,18 @@
 (rf/reg-event-fx
  :wallet-connect/fetch-persisted-sessions-success
  (fn [{:keys [db]} [sessions]]
-   (let [sessions' (mapv (fn [{:keys [sessionJson] :as session}]
-                           (assoc session
-                                  :accounts
-                                  (-> sessionJson
-                                      types/json->clj
-                                      :namespaces
-                                      :eip155
-                                      :accounts)))
-                         sessions)]
-     {:fx [[:dispatch [:wallet-connect/fetch-active-sessions]]]
+   (let [network-status (:network/status db)
+         sessions'      (mapv (fn [{:keys [sessionJson] :as session}]
```

Comment:
Do you really need the eager version of `map`?

Reply:
not my addition, but most likely not